week-6/Homework/mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,27 +1,118 @@
-function setAlarm() {}
+let timeLeft;
+let intervalId;
+let flashingIntervalId;
+let isPaused = false;
+let isRedBackground = false;
+let isBlueBackground = true;
```

Comment:
I find a bit confusing that 2 booleans, that allow for 4 combinations, are used to represent one of the 3 possible colors.

Can you think of a way to make it more intuitive?